python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -605,6 +612,8 @@ def process_graph(args):
     argparser = argparse.ArgumentParser("Preprocess graphs")
     argparser.add_argument("--conf-file", type=str, required=True,
                            help="The configuration file.")
+    argparser.add_argument("--output-conf-file", type=str,
```

Comment:
Default is None?

Reply:
by default, we don't output the new config file.